### PR TITLE
Doc for CCX

### PR DESCRIPTION
**Pull request by mhoeber**
This change:
- Removes CCX from Partner doc
- Includes edited CCX doc in open edx course staff doc
- Includes CCX config in install/config doc

@srpearce  can you review
@sarina

### CODE REVIEW

**Comment by sarina** [comment]
@mhoeber I don't know much about CCX, I've just been making sure that the project gets the resources it needs. If you need someone to review I recommend tagging the authors of #310

File: en_us/install_operations/source/front_matter/change_log.rst [hunk 1]
```diff
@@ -8,6 +8,8 @@ Change Log
 
    * - Date
      - Change
+   * - 16 June 2015
+     - Added the :ref:`Enable CCX` section
```
Comment (srpearce) [inline en_us/install_operations/source/front_matter/change_log.rst]: Add a period

File: en_us/shared/building_and_running_chapters/building_course/custom_courses.rst [hunk 2]
```diff
@@ -4,175 +4,166 @@
...
 **************************
- Enabling CCX for a Course
+Custom Courses and Roles
 **************************
 
-CCX for a course is disabled by default. It is enabled in Studio.
+There are three roles involved in creating and running a CCX. 
+
+* **Instructors** create the existing course and have privileges to create CCX
+  Coaches for their course.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/custom_courses.rst]: Do we need to capitalize coaches? I don't think we usually capitalize course roles (e.g., course staff). If we decide to lowercase, make the change throughout.

File: en_us/shared/building_and_running_chapters/building_course/custom_courses.rst [hunk 3]
```diff
@@ -4,175 +4,166 @@
...
+
+* **CCX Coaches** select content from the existing course for the CCX. CCX
+  Coaches also enroll learners, set start and due dates for their scheduled
+  content, and manage grading policy for the CCX.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/custom_courses.rst]: manage _the_ grading policy

File: en_us/shared/building_and_running_chapters/building_course/custom_courses.rst [hunk 4]
```diff
@@ -4,175 +4,166 @@
...
 *******************
- Adding a CCX Coach
+Add a CCX Coach
 *******************
 
-The first step to create a CCX is for an instructor to 
-create a CCX Coach in an existing course. At this time, a CCX can 
-have only one coach.
+The first step to create a CCX is to create a CCX Coach in an existing course.
+A CCX can have only one XXC coach.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/custom_courses.rst]: XXC --> CCX

**Comment by mhoeber** [comment]
@pwilkins  made some updates, we're ready to merge this

**Comment by pwilkins** [comment]
@mhoeber Where does the CCX doc appear now that it's merged?

**Comment by mhoeber** [comment]
@pwilkins sorry for delay.  this is live in open edx doc, not partner doc:  http://edx.readthedocs.org/projects/open-edx-building-and-running-a-course/en/latest/building_course/custom_courses.html#creating-a-custom-course

**Comment by pwilkins** [comment]
@mhoeber You removed a section from the CCX documentation.  Where did you move it?  It's critical that the section on CCX administration is available to CCX adopters.

**Comment by mhoeber** [comment]
@pwilkins   sorry, do you mean:  http://edx.readthedocs.org/projects/edx-installing-configuring-and-running/en/latest/configuration/enable_ccx.html

**Comment by pwilkins** [comment]
@mhoeber That's it!  Thank you.  I feared that section had gotten lost.

**Pull request merged**